src/turbolinks/script_warning.coffee

```diff
@@ -0,0 +1,16 @@
+do ->
```

Comment:
Each `.coffee` file has an implicit `do ->` so this adds an unnecessary second.

Reply:
Technically unnecessary, but I wanted to make it clear the three `return` statements short-circuit just the script check and not the whole Turbolinks boot process.